desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersViewModel.java

```diff
@@ -181,6 +183,11 @@ boolean isBootstrappedOrShowPopup() {
         return GUIUtil.isBootstrappedOrShowPopup(p2PService);
     }
 
+    public boolean isMakerFee(OpenOffer openOffer) {
+        Coin makerFee = openOffer.getOffer().getMakerFee();
+        return makerFee != null && makerFee.isPositive();
```

Comment:
`makerFee != null` check seems to be redundant

Reply:
Correct - I moved some code parts and didn't revisit if the null check is still necessary.